build.gradle

```diff
@@ -46,34 +46,34 @@ ext {
                                           exclude module: 'j2objc-annotations'
                                       }
                                   },
-            slf4j               : [group: 'org.slf4j', name: 'slf4j-api', version: '1.7.30'],
-            log4j_api           : [group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.17.2'],
-            log4j_core          : [group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.17.2'],
-            log4j_slf4j         : [group: 'org.apache.logging.log4j', name: 'log4j-slf4j-impl', version: '2.17.2'],
+            slf4j               : [group: 'org.slf4j', name: 'slf4j-api', version: '2.0.2'],
```

Comment:
Of course [`2.0.3`](https://github.com/qos-ch/slf4j/releases/tag/v_2.0.3) was released just ~4 h after I've added the change to this PR... 😂